libraries/rush-lib/src/logic/operations/PhasedOperationPlugin.ts

```diff
@@ -29,13 +29,18 @@ function createOperations(
   existingOperations: Set<Operation>,
   context: ICreateOperationsContext
 ): Set<Operation> {
-  const { projectsInUnknownState: changedProjects, phaseSelection, projectSelection } = context;
+  const {
+    projectsInUnknownState: changedProjects,
+    phaseOriginal,
+    phaseSelection,
+    projectSelection
+  } = context;
   const operationsWithWork: Set<Operation> = new Set();
 
   const operations: Map<string, Operation> = new Map();
 
   // Create tasks for selected phases and projects
-  for (const phase of phaseSelection) {
```

Comment:
This breaks how watch mode currently works. If we don't want to expand the set of phases, I'm fine with simply not expanding the set of selected phases and only tracking the dependencies when building out the graph.